src/Conversion/ONNXToTOSA/Math/Conv2D.cpp

```diff
@@ -136,8 +133,8 @@ class ONNXConvOpLoweringToTOSA : public OpConversionPattern<ONNXConvOp> {
     }
 
     // Convert output [N,H,W,M] -> [N,M,H,W]
```

Comment:
Aside: I'm curious to know where the convention of using `M` for output channels comes from?  
I'd adopted the `F` from linalg e.g. https://mlir.llvm.org/docs/Dialects/Linalg/#linalgconv_2d_nchw_fchw-mlirlinalgconv2dnchwfchwop and was indeed afraid there would be different traditions.